kong/db/schema/others/declarative_config.lua

```diff
@@ -1,5 +1,6 @@
 local uuid = require("resty.jit-uuid")
-local utils = require("kong.tools.utils")
+local kong_uuid = require("kong.tools.uuid").uuid
```

Comment:
This line may be conflict with #12952, could we keep it?

Reply:
yes